src/modules/Lan.rb

```diff
@@ -214,51 +214,27 @@ def isAnyInterfaceDown
       down
     end
 
+    # Checks local configuration if IPv6 is allowed
+    #
+    # return [Boolean] true when IPv6 configuration is loaded properly
```

Comment:
"true when IPv6 configuration is loaded properly" is ambiguous. I understood it as "true if the configuration settings have been read without error, and IPv6 may be on or off" but it in fact means "true if IPv6 is enabled".